code/modules/food_and_drinks/recipes/soup_mixtures.dm

```diff
@@ -0,0 +1,1720 @@
+	initial_reagent = /datum/reagent/consumable/nutriment/soup/hotchili
+
+/datum/chemical_reaction/food/soup/hotchili
+	required_reagents = list(/datum/reagent/water = 50)
+	required_ingredients = list(
+		/obj/item/food/meat/cutlet = 2,
+		/obj/item/food/grown/chili = 1,
+		/obj/item/food/grown/tomato = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/hotchili = 30,
+		/datum/reagent/consumable/tomatojuice = 10,
+		// Capsaicin comes from the chillis, meaning you can make tame chili.
+		/*
+		/datum/reagent/consumable/nutriment = 4,
+		/datum/reagent/consumable/nutriment/protein = 6,
+		/datum/reagent/consumable/nutriment/vitamin = 4
+		*/
+	)
+	ingredient_reagent_multiplier = 0.33 // Chilis have a TON of capsaicin naturally
+	percentage_of_nutriment_converted = 0
+
+// Chili (Cold)
+/datum/reagent/consumable/nutriment/soup/coldchili
+	name = "Cold Chili"
+	description = "This slush is barely a liquid!"
+	data = list("tomato" = 1, "mint" = 1)
+	glass_price = FOOD_PRICE_NORMAL
+	color = "#3861C2"
+
+/datum/glass_style/has_foodtype/soup/coldchili
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/coldchili
+	icon_state = "coldchili"
+	drink_type = VEGETABLES | MEAT
+
+/datum/chemical_reaction/food/soup/coldchili
+	required_reagents = list(/datum/reagent/water = 50)
+	required_ingredients = list(
+		/obj/item/food/meat/cutlet = 2,
+		/obj/item/food/grown/icepepper = 1,
+		/obj/item/food/grown/tomato = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/coldchili = 30,
+		/datum/reagent/consumable/tomatojuice = 10,
+		// Frost Oil comes from the chilis
+	)
+	ingredient_reagent_multiplier = 0.33 // Chilis have a TON of frost oil naturally
+	percentage_of_nutriment_converted = 0
+
+// Chili (Clownish)
+/datum/reagent/consumable/nutriment/soup/clownchili
+	name = "Chili Con Carnival"
+	description = "A delicious stew of meat, chiles, and salty, salty clown tears."
+	data = list(
+		"tomato" = 1,
+		"hot peppers" = 2,
+		"clown feet" = 2,
+		"kind of funny" = 2,
+		"someone's parents" = 2,
+	)
+	glass_price = FOOD_PRICE_EXOTIC
+	color = "#FF0000"
+
+/datum/glass_style/has_foodtype/soup/clownchili
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/clownchili
+	icon_state = "clownchili"
+	drink_type = VEGETABLES | MEAT
+
+/datum/chemical_reaction/food/soup/clownchili
+	required_reagents = list(/datum/reagent/water = 50)
+	required_ingredients = list(
+		/obj/item/food/meat/cutlet = 2,
+		/obj/item/food/grown/chili = 1,
+		/obj/item/food/grown/tomato = 1,
+		/obj/item/clothing/shoes/clown_shoes = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/clownchili = 30,
+		/datum/reagent/consumable/tomatojuice = 8,
+		/datum/reagent/consumable/laughter = 4,
+		/datum/reagent/consumable/banana = 4,
+	)
+	percentage_of_nutriment_converted = 0.15
+
+// Vegan Chili
+/datum/reagent/consumable/nutriment/soup/chili_con_carne
+	name = "Chili Sin Carne"
```

Comment:
man this type path sucks for what it really is "chili con carne" path for "chili _sin_ carne"? come on. it made me think you somehow screwed up the "chili con carnival" but that has a `/clownchili` typepath so whatever

no need to change it but i wnated to mentoin it now before i forget